data/sprites/sprites.asm

```diff
@@ -1,360 +1,80 @@
-	dw NurseSprite
-	db $c0 ; byte count
-	db BANK(NurseSprite)
-
-	; SPRITE_CABLE_CLUB_WOMAN
-	dw CableClubWomanSprite
-	db $c0 ; byte count
-	db BANK(CableClubWomanSprite)
-
-	; SPRITE_MR_MASTERBALL
-	dw MrMasterballSprite
-	db $c0 ; byte count
-	db BANK(MrMasterballSprite)
-
-	; SPRITE_LAPRAS_GIVER
-	dw LaprasGiverSprite
-	db $c0 ; byte count
-	db BANK(LaprasGiverSprite)
-
-	; SPRITE_WARDEN
-	dw WardenSprite
-	db $c0 ; byte count
-	db BANK(WardenSprite)
-
-	; SPRITE_SS_CAPTAIN
-	dw SsCaptainSprite
-	db $c0 ; byte count
-	db BANK(SsCaptainSprite)
-
-	; SPRITE_FISHER2
-	dw Fisher2Sprite
-	db $c0 ; byte count
-	db BANK(Fisher2Sprite)
-
-	; SPRITE_BLACKBELT
-	dw BlackbeltSprite
-	db $c0 ; byte count
-	db BANK(BlackbeltSprite)
-
-	; SPRITE_GUARD
-	dw GuardSprite
-	db $c0 ; byte count
-	db BANK(GuardSprite)
-
-	; $32
-	dw GuardSprite
-	db $c0 ; byte count
-	db BANK(GuardSprite)
-
-	; SPRITE_MOM
-	dw MomSprite
-	db $c0 ; byte count
-	db BANK(MomSprite)
-
-	; SPRITE_BALDING_GUY
-	dw BaldingGuySprite
-	db $c0 ; byte count
-	db BANK(BaldingGuySprite)
-
-	; SPRITE_YOUNG_BOY
-	dw YoungBoySprite
-	db $c0 ; byte count
-	db BANK(YoungBoySprite)
-
-	; SPRITE_GAMEBOY_KID
-	dw GameboyKidSprite
-	db $c0 ; byte count
-	db BANK(GameboyKidSprite)
-
-	; SPRITE_GAMEBOY_KID_COPY
-	dw GameboyKidSprite
-	db $c0 ; byte count
-	db BANK(GameboyKidSprite)
-
-	; SPRITE_CLEFAIRY
-	dw ClefairySprite
-	db $c0 ; byte count
-	db BANK(ClefairySprite)
-
-	; SPRITE_AGATHA
-	dw AgathaSprite
-	db $c0 ; byte count
-	db BANK(AgathaSprite)
-
-	; SPRITE_BRUNO
-	dw BrunoSprite
-	db $c0 ; byte count
-	db BANK(BrunoSprite)
-
-	; SPRITE_LORELEI
-	dw LoreleiSprite
-	db $c0 ; byte count
-	db BANK(LoreleiSprite)
-
-	; SPRITE_SEEL
-	dw SeelSprite
-	db $c0 ; byte count
-	db BANK(SeelSprite)
-
-	; SPRITE_BALL
-	dw BallSprite
-	db $40 ; byte count
-	db BANK(BallSprite)
-
-	; SPRITE_OMANYTE
-	dw OmanyteSprite
-	db $40 ; byte count
-	db BANK(OmanyteSprite)
-
-	; SPRITE_BOULDER
-	dw BoulderSprite
-	db $40 ; byte count
-	db BANK(BoulderSprite)
-
-	; SPRITE_PAPER_SHEET
-	dw PaperSheetSprite
-	db $40 ; byte count
-	db BANK(PaperSheetSprite)
-
-	; SPRITE_BOOK_MAP_DEX
-	dw BookMapDexSprite
-	db $40 ; byte count
-	db BANK(BookMapDexSprite)
-
-	; SPRITE_CLIPBOARD
-	dw ClipboardSprite
-	db $40 ; byte count
-	db BANK(ClipboardSprite)
-
-	; SPRITE_SNORLAX
-	dw SnorlaxSprite
-	db $40 ; byte count
-	db BANK(SnorlaxSprite)
-
-	; SPRITE_OLD_AMBER_COPY
-	dw OldAmberSprite
-	db $40 ; byte count
-	db BANK(OldAmberSprite)
-
-	; SPRITE_OLD_AMBER
-	dw OldAmberSprite
-	db $40 ; byte count
-	db BANK(OldAmberSprite)
-
-	; SPRITE_LYING_OLD_MAN_UNUSED_1
-	dw LyingOldManSprite
-	db $40 ; byte count
-	db BANK(LyingOldManSprite)
-
-	; SPRITE_LYING_OLD_MAN_UNUSED_2
-	dw LyingOldManSprite
-	db $40 ; byte count
-	db BANK(LyingOldManSprite)
-
-	; SPRITE_LYING_OLD_MAN
-	dw LyingOldManSprite
-	db $40 ; byte count
-	db BANK(LyingOldManSprite)
+SpriteSheetPointerTable:
+; graphics, tile count
+	overworld_sprite RedSprite, 12 ; SPRITE_RED
```

Comment:
Align these comments perhaps?

Reply:
I don't think there's been any convention for whether this sort of comment should be indented or not, but sure, I can indent them.

Reply:
Actually, a lot of this sort of comment should be revised to have comments linking constants and data together, so I'll do that separately.